CONTRIBUTING.md

```diff
@@ -1,26 +1,42 @@
+- Don't use the issue tracker for support questions.
+- Consider [enabling debug mode](https://github.com/laurent22/joplin/blob/master/readme/debugging.md) before so that you can provide as much details as possible when reporting the issue.
+- Stay on topic, but describe the issue in detail so that others can reproduce it.
+- **Provide a screenshot** if possible. A screenshot showing the problem is often more useful than a paragraph describing it.
 
 # Feature requests
 
-Again, please check that it has not already been requested. If it has, simply **up-vote the issue** - the ones with the most up-votes are likely to be implemented. "+1" comments are not tracked.
+Please check that your request has not already been posted in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). If it has, **up-voting the issue** increases the chances it'll be noticed and implemented in the future. "+1" comments are not tracked.
 
-# Creating a pull request
+As a general rule, suggestions to _improve Joplin_ should be posted first in the [Joplin Forum](https://discourse.joplin.cozic.net/) for discussion.
 
-- If you want to add a new feature, consider asking about it before implementing it or checking existing discussions to make sure it is within the scope of the project. As a rule of thumb **if your change is likely to involve more than 50 lines of code, you should discuss it in the forum**, just so that you don't spend too much time implementing something that might not be accepted.
+Avoid listing multiple requests in one report in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue). One issue per request makes it easier to track and discuss it.
 
-- Bug fixes are always welcome.
+# Contribute to the project
+
+## Contributing to Joplin's translation
+
+Joplin is available in multiple languages thanks to the help of its users. You can help translate Joplin to your language or keep it up to date. Please read the documentation about [Localisation](https://github.com/laurent22/joplin#localisation).
+
+## Contributing to Joplin's code
+
+If you want to start contributing to the project's code, please follow these guidelines before creating a pull request: 
+
+- Bug fixes are always welcome. Start by reviewing the list of [essential issues](https://github.com/laurent22/joplin/issues?q=is%3Aissue+is%3Aopen+label%3Aessential)
+- Before adding a new feature, ask about it in the [Github Issue Tracker](https://github.com/laurent22/joplin/issues?utf8=%E2%9C%93&q=is%3Aissue) or the [Joplin Forum](https://discourse.joplin.cozic.net/), or check if existing discussions exist to make sure the new functionality is desired.
+- **Changes that will consist in more than 50 lines of code should be discussed the [Joplin Forum](https://discourse.joplin.cozic.net/)**, so that you don't spend too much time implementing something that might not be accepted.
```

Comment:
Changes that consist of more than 50 lines of code should be discussed in the ...

new Joplin Forum address: https://discourse.joplinapp.org/